php.ini

```diff
@@ -1,4 +1,38 @@
 extension_dir=/opt/lib/php/7.1/modules
+display_errors=On
+
+;extension=bz2.so
+;extension=calendar.so
+;extension=ctype.so
 extension=curl.so
+;extension=dom.so
+;extension=exif.so
+;extension=fileinfo.so
+;extension=ftp.so
+;extension=gettext.so
+;extension=iconv.so
 extension=json.so
-display_errors=On
\ No newline at end of file
+;extension=mbstring.so
+;extension=mysqli.so
+;extension=mysqlnd.so
+;extension=pdo.so
+;extension=pdo_mysql.so
+;extension=pdo_pgsql.so
+;extension=pdo_sqlite.so
+;extension=pgsql.so
+;extension=phar.so
+;extension=posix.so
+;extension=shmop.so
+;extension=simplexml.so
+;extension=sockets.so
+;extension=sqlite3.so
+;extension=sysvmsg.so
+;extension=sysvsem.so
+;extension=sysvshm.so
+;extension=tokenizer.so
+;extension=wddx.so
+;extension=xml.so
+;extension=xmlreader.so
+;extension=xmlwriter.so
+;extension=xsl.so
+;extension=zip.so
```

Comment:
OK, I can remove them. The basic idea was to give a template for people creating functions to just copy this php.ini and uncomment the modules they need, rather than looking into the documentation. No problem to remove them, anyhow.